test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/MaxRequestBufferSizeTests.cs

```diff
@@ -55,7 +55,7 @@ public static IEnumerable<object[]> LargeUploadData
                     Tuple.Create((long?)_dataLength - 1, false),
 
                     // Buffer is exactly the same size as data.  Exposed race condition where
-                    // IConnectionControl.Resume() was called after socket was disconnected.
+                    // ITimeoutControl.Resume() was called after socket was disconnected.
```

Comment:
This happened because the first draft of this renamed `ITimeoutControl => IConnectionControl`. Reverting that change found this. Not sure what this is supposed to be. We don't have an IConnectionControl in dev or this PR.